recordedit/form.controller.js

```diff
@@ -7,7 +7,7 @@
         var vm = this;
         var context = $rootScope.context;
         vm.recordEditModel = recordEditModel;
-        vm.editMode = context.filter || false;
+        vm.editMode = (context.filter && !context.copy) || false;
```

Comment:
The `|| false` is probably unnecessary here. `context.filter && !context.copy` will evaluate to `false` if it's not `true`.